### DISCUSSION TITLE
Is there a way to convert default payload format to json in a transparent way?

### DISCUSSION THREAD

**Discussion by sombriks** [bigskysoftware/htmx#2325] [Q&A] [answered]
most of REST APIS are expecting json payload and a good amount of tools exists to handle it.

how could i make htmx payloads to send json instead form?

does it make any sense?

**Comment by nickchomey** [comment, marked as answer]
No, I don't think this makes any sense at all. Htmx works with hypermedia, not json. Also, rest APIs that work with json aren't actually rest APIs - they're json APIs.

If you don't want to work with hypermedia, just use the "good amount of tools" that expect json rather than htmx...

Check out https://htmx.org/essays to learn more about all of this

**Reply by sombriks** [reply]
yes that's correct.
i need to rewire some neural paths, thanks for the guidance!

**Reply by nickchomey** [reply]
No problem. Yeah, it generally takes some time to re-program ourselves after a decade of enforced malpractice.
